vendor/brave-ios/Ads/BATAdsNotification.mm

```diff
@@ -3,34 +3,34 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #import "BATAdsNotification.h"
-#include "bat/ads/notification_info.h"
+#include "bat/ads/ad_notification_info.h"
 #import <map>
 
 @interface BATAdsNotification ()
-@property (nonatomic, copy) NSString *id;
+@property (nonatomic, copy) NSString *idterry;
+@property (nonatomic, copy) NSString *creativeInstanceId;
 @property (nonatomic, copy) NSString *creativeSetID;
 @property (nonatomic, copy) NSString *category;
-@property (nonatomic, copy) NSString *advertiser;
-@property (nonatomic, copy) NSString *text;
-@property (nonatomic, copy) NSURL *url;
-@property (nonatomic, copy) NSString *uuid;
+@property (nonatomic, copy) NSString *title;
+@property (nonatomic, copy) NSString *body;
+@property (nonatomic, copy) NSURL *target_url;
```

Comment:
Changes made here to `readwrite` overrides of the properties but not the public header (`BATAdsNotification.h`) properties

Reply:
Also wrong style for Obj-C `target_url` → `targetURL`

Reply:
Fixed